Retail upgradepath more 41 keichwa (#148)
uyuni-project/uyuni-docs: modules/retail/pages/retail-upgrade-branchserver.adoc

[[retail-upgrade-branchserver]]
= Upgrade {smr} Branch Server

This section describes upgrading a {smr} Branch Server to the next SP (service pack).

{smr} Branch Server is a client system such as a {productname} Proxy with additional {smr} features.

> For more information about upgrading a proxy client, see xref:upgrade:proxy-update.adoc[].

Reviewer: A note that `Highstate should be applied on migrated branch/proxy to update retail functionality` should be here.
Just updating packages will not update retail configurations (i.e. new http mode will not be enabled by package update). Assuming that suma server is updated first.

Author: ok, thanks.  I hope I got it right.

This section describes upgrading a {smr} Branch Server to the next SP (service pack).

{smr} Branch Server is a client system such as a {productname} Proxy with additional {smr} features.

[NOTE]
====

> Upgrade the {productname} Server before beginning with the {srm} upgrade.

Reviewer 2: ```suggestion
Upgrade the {productname} Server before starting the {smr} upgrade.
```

====

> 

Reviewer 2: ```suggestion
```

Author: I think we want 3 empty lines before a procedure title ;)

.Procedure: Upgrading the {smr} Branch Server
. For general information about upgrading a proxy client, see xref:upgrade:proxy-update.adoc[].

> . On the {smr} Branch Server apply highstate after the general proxy upgrade finished successfully.

Reviewer 2: ```suggestion
. After the proxy upgrade is complete, apply the highstate on the {smr} Branch Server.
```
